libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -125,6 +125,8 @@ bool FunctionCallGraphBuilder::visit(FunctionCall const& _functionCall)
 		// change at runtime). All we can do is to add an edge to the dispatch which in turn has
 		// edges to all functions could possibly be called.
 		add(m_currentNode, CallGraph::SpecialNode::InternalDispatch);
+	else if (functionType->kind() == FunctionType::Kind::Error)
+		m_graph.usedErrors.insert(&dynamic_cast<ErrorDefinition const&>(functionType->declaration()));
```

Comment:
maybe we should add an assert here?

Reply:
You mean as `else`? Not really. There are other case possible here, the graph just skips them.

Reply:
There is a check and I would say it is highly unlikely for this assert to trigger...